utilities_common/auto_techsupport_helper.py

```diff
@@ -0,0 +1,172 @@
+import os
+import re
+import sys
+import glob
+import time
+import argparse
+import subprocess
+import shutil
+import math
+import syslog
+from os.path import basename, splitext
+
+# MISC
+CORE_DUMP_DIR = "/var/core"
+CORE_DUMP_PTRN = "*.core.gz"
+
+TS_DIR = "/var/dump"
+TS_PTRN = "sonic_dump_*.tar*"
+
+# CONFIG DB Attributes
+CFG_DB = "CONFIG_DB"
```

Comment:
All these attributes after this statement belong to CONFIG_DB. 
```
# AUTO_TECHSUPPORT|GLOBAL table attributes
AUTO_TS = "AUTO_TECHSUPPORT|GLOBAL"
CFG_STATE = "state"
CFG_MAX_TS = "max_techsupport_size"
COOLOFF = "rate_limit_interval"
CFG_CORE_USAGE = "max_core_size"
CFG_SINCE = "since"

# AUTO_TECHSUPPORT_FEATURE Table
FEATURE = "AUTO_TECHSUPPORT_FEATURE|{}"
```

and thus added STATE_DB = "STATE_DB" after this